doc/help/elektra-cmerge-strategy.md

```diff
@@ -0,0 +1,32 @@
+# elektra-cmerge-strategies(7) -- how to merge key sets
+
+## 3-WAY
+
+- `base`:
+  The `base` KeySet is the original version of the KeySet.
```

Comment:
What is a KeySet and what is the `original version`?

Reply:
Shouldn't everyone who comes to a document about merging keysets already know the basics of Elektra and therefore know what a key set is?

Reply:
People reading only the man pages might not have come across KeySets.